### DISCUSSION TITLE
Session logout after redirecting from other site

### DISCUSSION THREAD

**Discussion by hatchesping** [codeigniter4/shield#980] [Q&A]
Hi, I'm integrating Shield into one of my client's project and all is fine until I redirect the user to payment gateway website to make my payment, then from payment gateway redirecting back to the website the user will get logout automatically, I think it's because the payment gateway POST some payment data to the website and causing this, so I'm not sure is there any setting I can set to prevent this from happening? Thanks.

**Comment by olajideolamide** [comment]
This issue is because of the CORS policy. 
And you are correct, this happens because of the POST request.

It has already been discussed here: https://forum.codeigniter.com/showthread.php?tid=82384

**Reply by hatchesping** [reply]
Just found out that yesterday, thanks alot!
